src/openfermion/utils/_trotter_exp_to_qgates.py

```diff
@@ -0,0 +1,277 @@
+#   Licensed under the Apache License, Version 2.0 (the "License");
+#   you may not use this file except in compliance with the License.
+#   You may obtain a copy of the License at
+#
+#       http://www.apache.org/licenses/LICENSE-2.0
+#
+#   Unless required by applicable law or agreed to in writing, software
+#   distributed under the License is distributed on an "AS IS" BASIS,
+#   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#   See the License for the specific language governing permissions and
+#   limitations under the License.
+"""Module to perform Trotter-Suzuki decompositions to output as circuits."""
+
+from openfermion.ops import QubitOperator
+import numpy
+import copy
+import collections
+
+"""
+Description:
+    Functions for estimating the exponential of an operator
+    composed of Pauli matrices, by Trotterization. Outputs QASM
+    format to a python stream.
+
+    Change-of-basis gates:
+    H           : Z to X basis (and back)
+    Rx(pi/2)    : Z to Y basis .....
+    Rx(-pi/2)   : Y to Z basis .....
+"""
+
+
+
+def _third_order_trotter_helper(op_list):
+    """Iteratively trotterize the evolution under a QubitOperator according
+        to the scheme: e^(A+B) = e^(7/24 * A) e^(2/3 * B) e^(3/4 * A)
+        e^(-2/3 * B) e^(-1/24 * A) e^(B)
+
+    Note:
+        See N. Hatano and M. Suzuki Lect. Notes Phys 679, 37-68 (2005)
+
+    Args:
+        op_list: (list of QubitOperator) the terms in the
```

Comment:
Type before colon, try to match http://sphinxcontrib-napoleon.readthedocs.io/en/latest/example_google.html when possible